flowmeter/src/utils.ts

```diff
@@ -0,0 +1,51 @@
+import { CamOverlayDrawingAPI, UploadImageResponse } from 'camstreamerlib/CamOverlayDrawingAPI';
+import * as fs from 'fs';
+
+export class MemoryManager {
+    private imgFiles: Record<string, string> = {};
```

Comment:
maybe the name of file should be MemoryManager instead of utils

Reply:
solved